salt/fileclient.py

```diff
@@ -1169,15 +1169,7 @@ def get_file(
         if senv:
             saltenv = senv
 
-        if not salt.utils.platform.is_windows():
-            hash_server, stat_server = self.hash_and_stat_file(path, saltenv)
-            try:
-                mode_server = stat_server[0]
-            except (IndexError, TypeError):
-                mode_server = None
```

Comment:
It's not clear why but pylint doesn't appear to be warning that these are set but never read